### DISCUSSION TITLE
make variables available also in terminal invokation

### DISCUSSION THREAD

**Discussion by jmlucjav** [doublecmd/doublecmd#1615] [Suggestions]
I want to start the terminal (F9) in the current dir. Could not find a way.

I would like it to open in the current dir. If we could use https://doublecmd.github.io/doc/en/variables.html that would be possible for most terminals, as they have params for this.

thanks

**Comment by alexx2000** [comment]
It starts in the current directory by default. What terminal program do you use? You can use `%D` variable in the [terminal settings](https://doublecmd.github.io/doc/en/configuration.html#ConfigToolsTerminal).

**Comment by jmlucjav** [comment]
I'm using wezterm, I managed to do it with:

```
command: wezterm-gui
params: --config-file myfile.lua start --cwd %D
```

if I don't supply the %D, it won't start in current path. 
Thanks
